running/ambassador-edge/aes.yaml

```diff
@@ -203,14 +203,13 @@ metadata:
     product: aes
     app.kubernetes.io/component: ambassador-service
 spec:
-  type: LoadBalancer
   ports:
     - name: http
       port: 80
-      targetPort: http
+      targetPort: 8080
```

Comment:
?? you just keeping Ambassador around on 8080 for a bit just in case Nginx doesn't work out?

Reply:
This actually doesn’t change anything, was me just trying to fix stuff.